### PR TITLE
#167690864 Create user dropdown menu

### PR DESCRIPTION
**Pull request by kechyy**
#### Description
 - This PR creates user dropdown menu for desktop and mobile view

#### Type of Change
- [x] Feature (Create user dropdown menu)

#### How Has This Been Tested?
- [x] Clone the repo [ErrorSwag](https://github.com/andela/ah-kifaru-frontend)
- [x] Run npm install on the terminal
- [x] run npm start

#### Checklist:
- [x] Create user dropdown menu for desktop view
- [x] Create user dropdown menu for mobile view

#### Pivotal Stories
[#167690864](https://www.pivotaltracker.com/story/show/167690864)

### CODE REVIEW

**Review by Onyimatics — CHANGES_REQUESTED** [review]
Good job @kechyy and @Onyimatics . I know everything is working out fine on this PR. But then, we need to fix the continuous integration part of this PR.

**Review by timi-codes — CHANGES_REQUESTED** [review]
Good Job @kechyy @Onyimatics , Kindly fix your Travis and Circle CI build

**Comment by Onyimatics** [comment]
> Good Job @kechyy @Onyimatics , Kindly fix your Travis and Circle CI build

Thank you @timi-codes . We will do that.

**Comment by kechyy** [comment]
> > Good Job @kechyy @Onyimatics , Kindly fix your Travis and Circle CI build
> 
> Thank you @timi-codes . We will do that.

Thank you @timi-codes for the feedback, we will work on that.

File: tests/App.spec.js [hunk 1]
```diff
@@ -28,4 +22,8 @@ describe('Application test', () => {
 
     expect(toJson(wrapper)).toMatchSnapshot();
   });
+  it('should render without crashing', () => {
+    const wrapper = shallow(<Menu />);
```
Comment (segunolalive) [inline tests/App.spec.js]: Mount the component using Enzyme's `mount` method.

Also, assert that the dropdown contains expected DOM nodes e.g what links are rendered and their `href` etc.

File: src/components/userMenu/Menu.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@media only screen and (max-width: 640px) {
```
Comment (segunolalive) [inline src/components/userMenu/Menu.css:1]: use SCSS. Define SCSS variables for screen sizes, colours.

File: src/components/userMenu/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import './Menu.css';
+
+class Menu extends Component {
```
Comment (segunolalive) [inline src/components/userMenu/index.js]: This component does not use state nor have any side effects. Do convert to a function component.

**Review by segunolalive — CHANGES_REQUESTED** [review]
Good work. Make the changes and **Merge**

File: tests/App.spec.js (hunk 1 above)
Comment (Onyimatics) [reply tests/App.spec.js]: Okay, we will do that. Thank you for the feedback.

File: src/components/userMenu/Menu.css (hunk 2 above)
Comment (Onyimatics) [reply src/components/userMenu/Menu.css:1]: Okay.

File: src/components/userMenu/index.js (hunk 3 above)
Comment (Onyimatics) [reply src/components/userMenu/index.js]: All right.

**Pull request merged**